### PR TITLE
Release/4 0 1

### PR DESCRIPTION
**Pull request by bricesanchez**
@parndt Could you release 4.0.1 without #3289? it will be better to release it for 4.1.0

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
+## 4.1.0 (unreleased)
+
+* Separate Dragonfly from Images and Resources. [#3289](https://github.com/refinery/refinerycms/pull/3289). [Anita Graham](https://github.com/anitagraham)
+
+* [See full list](https://github.com/refinery/refinerycms/compare/4-0-stable...4.1.0)
+
+## 4.0.1 [unreleased]
```
Comment (parndt) [inline changelog.md]: If you set this to `20 October 2017` then I'll release it today - also we need this PR for `4-0-stable` branch without the `4.1.0` stuff

**Comment by parndt** [comment]
@bricesanchez we'll need this changelog update for `4-0-stable` too

**Comment by bricesanchez** [comment]
Done https://github.com/refinery/refinerycms/commits/4-0-stable

**Comment by parndt** [comment]
Released 👍

**Pull request merged**